### ISSUE TITLE
Allow specifying a different output directory than app/?

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [GoogleChromeLabs/bubblewrap#576]
**Is your feature request related to a problem? Please describe.**
I'm trying to use bubblewrap on an existing project that, per Rails conventions, has `.rb` files in folders under `app/` such as `app/models`. This conflicts with the default bubblewrap build outputs under `app/build` and `app/src`.

**Describe the solution you'd like**
Can we have a flag like `--outDir` to output to a different location?

```shell
bubblewrap build --outDir built
```

**Describe alternatives you've considered**
Moving the original `app/` to a temp dir before build, then moving the build `app/` to another temp folder, and moving the original `app/` contents back. I don't like this 😄

**Comment by andreban** [comment]
You should be able to modify your `gradle.properties` and specify a different build directory, as described here: https://stackoverflow.com/questions/13915351/change-gradles-working-directory-when-compiling-a-groovy-project

We could add this to `twa-manifest.json` so that `gradle.properties` is generated with the correct value when the project is regenerated.

**Comment by JoshuaKGoldberg** [comment]
+1, having to dive into Gradle for a web project is not something I'd expect to have to do 😉

**Comment by andreban** [comment]
The goal is that you shouldn't have to. 

Generally, you shouldn't have to care about the `app` folder too. I wonder if the reason is that you are using it in an existing project? Can you provide more details?

**Comment by JoshuaKGoldberg** [comment]
Yup! It's an existing Rails project that has `app/`, `config/`, and `lib/` directories for Ruby code, alongside a `webpack/` directory for frontend code.

**Comment by joshua-holmes** [comment]
My solution was to create a custom npm script in the package.json. I could cd into another directory other than app/ and then run the command. It would look something like this:
```
"scripts": {
    "build-android": "cd ./android && bubblewrap init ..."
}
```

**Comment by JoshuaKGoldberg** [comment]
Closing out my old issues I no longer have context on. If anybody else is still seeing this, I'd encourage you to file a new issue with more information. Cheers!
